doc/configcommands.dsv

```diff
@@ -8,7 +8,7 @@ bind-key||<key> <operation> [<dialog>]||n/a||Bind key <key> to <operation>. This
 bookmark-autopilot||[yes/no]||no||If set to `yes`, the configured bookmark command is executed without any further input asked from user, unless the url or the title cannot be found/guessed.||bookmark-autopilot yes
 bookmark-cmd||<command>||""||If set, then <command> will be used as bookmarking plugin. See the documentation on bookmarking for further information.||bookmark-cmd "~/bin/delicious-bookmark.sh"
 bookmark-interactive||[yes/no]||no||If set to `yes`, then the configured bookmark command is an interactive program.||bookmark-interactive yes
-browser||<command>||%BROWSER, otherwise lynx||Set the browser command to use when opening an article in the browser. If the <<BROWSER,`BROWSER`>> environment variable is set, it will be used as the default browser, otherwise lynx will be used. Any occurrences of `%u` in <command> will be replaced by the URL being opened, enclosed in single quotes. Any occurrences of `%F` in <command> will be replaced by the feed's URL in single quotes.||browser "w3m %u"
+browser||<command>||%BROWSER, otherwise lynx||Set the browser command to use when opening an article in the browser. If the <<BROWSER,`BROWSER`>> environment variable is set, it will be used as the default browser, otherwise lynx will be used. Any occurrences of `%u` in <command> will be replaced by the URL being opened, enclosed in single quotes. Any occurrences of `%F` in <command> will be replaced by the feed's URL in single quotes. Any occurences of `%t` in <command> will be replaced by the type of the URL being opened, enclosed in single quotes.||browser "w3m %u"
```

Comment:
This paragraph is getting too long. Also, the values for `%t` should be documented. I suggest adding a table under the "Open Links with External Commands" section, and adding a paragraph about `%F` and `%t` in there. Then, the last three sentences here can be replaced by a reference to that doc section. What do you think?